includes/Admin/SetupWizard.php

```diff
@@ -774,10 +735,22 @@ public function dokan_setup_recommended() {
     public function dokan_setup_recommended_save() {
         check_admin_referer( 'dokan-setup' );
 
+        $setup_store_growth           = isset( $_POST['setup_store_growth'] ) && 'yes' === sanitize_text_field( wp_unslash( $_POST['setup_store_growth'] ) );
         $setup_wc_conversion_tracking = isset( $_POST['setup_wc_conversion_tracking'] ) && 'yes' === sanitize_text_field( wp_unslash( $_POST['setup_wc_conversion_tracking'] ) );
         $setup_wemail                 = isset( $_POST['setup_wemail'] ) && 'yes' === sanitize_text_field( wp_unslash( $_POST['setup_wemail'] ) );
         $setup_texty                  = isset( $_POST['setup_texty'] ) && 'yes' === sanitize_text_field( wp_unslash( $_POST['setup_texty'] ) );
 
+        if ( $setup_store_growth && ! $this->is_store_growth_active() ) {
```

Comment:
- Pls also refactor this section so that we can handle it dynamically.
- Check the `current_user_can( 'manage_woocommerce' )`  before the install.